pages/change-password/_token.vue

```diff
@@ -67,7 +82,7 @@ export default defineComponent({
     }))
     const router = useRouter()
     onDone(() => {
-      void router.push({ name: 'user-login' })
+      void router.push('../user/login')
```

Comment:
Is this change really necessary? (It's definitely unrelated to the actual purpose of this PR)